stdlib/websub/src/main/ballerina/websub/Module.md

```diff
@@ -10,40 +10,86 @@ This implementation supports introducing all WebSub components: subscribers, pub
 is updated by the topic's publisher.
  
  
-#### Basic Flow with WebSub
+### Basic Flow with WebSub
 1. Subscriber discovers the topics it needs to subscribe to in order to receive updates/content and discovers the hub(s) where it can subscribe.
 
 2. Subscriber sends a subscription request to a hub, specifying the topic it needs to receive notifications for along 
  with other subscription parameters, such as:
-      - The callback URL where content is expected to be delivered
-      - (Optional) A lease seconds value indicating how long the subscriber wants the subscription to stay active
-      - (Optional) A secret to use for [authenticated content distribution](https://www.w3.org/TR/websub/#signing-content)
+    - The callback URL where content is expected to be delivered
+    - (Optional) A lease seconds value indicating how long the subscriber wants the subscription to stay active
+    - (Optional) A secret to use for [authenticated content distribution](https://www.w3.org/TR/websub/#signing-content)
   
-3. The hub sends an intent verification request to the specified callback URL, and if the response indicates verification
+3. The hub sends an intent verification request to the specified callback URL, and if the response indicates 
+verification
  (by echoing a challenge specified in the request) by the subscriber, the subscription is added for the topic at the 
  hub.
    
 4. Publisher notifies the hub of updates to the topic, and the content to deliver is identified.
 
 5. The hub delivers the identified content to the subscribers of the topic.
 
-#### Features
+### Features
+
+#### Subscriber
+
 This module allows introducing a WebSub Subscriber Service with `onIntentVerification`, which accepts HTTP GET requests for intent verification, and `onNotification`, which accepts HTTP POST requests for notifications. The WebSub Subscriber Service provides the following capabilities:
  - Subscription Requests are sent at service start time for the hub and topic, which are either specified as annotations or discovered based on the resource URL specified as an annotation.
  - Auto Intent Verification against the topic specified as an annotation, or discovered based on the resource URL specified as an annotation, if `onIntentVerification` is not specified.
  - Signature Validation for authenticated content distribution if a secret is specified for the subscription.
   
+#### Hub
+
 A WebSub compliant hub based on the Ballerina Message Broker is also available for use as a remote hub or to be used by publishers who want to have their own internal hub. Ballerina's WebSub Hub honors specified lease periods and supports authenticated content distribution.
 
+##### Enabling Data Persistence for the Hub
+
+The Ballerina WebSub Hub supports persistence of topic and subscription data that needs to be restored when the hub is 
+restarted.
+ 
+Users can introduce their own persistence implementation, by introducing an object type structurally 
```

Comment:
```suggestion
You can introduce your own persistence implementation by introducing an object type that is structurally 
```

Reply:
I believe we try to avoid active voice in Module.mds. Sharing an old comment from Sam - https://github.com/ballerina-platform/ballerina-lang/pull/10950#discussion_r230635825. Will do the other change.